src/warm-cache-for-festival.ts

```diff
@@ -143,12 +152,10 @@ async function warm(festival: string, years: number[]) {
                     }
                 }
             }
-        }
 
-        // Save all days so we can pull them later
-        redisClient.set(`festival:${festival.toLowerCase()}_${year}:days`,
-                        JSON.stringify(Object.keys(artistObjs)),
-                        redis.print);
+            // Save all days so we can pull them later
+            redisClient.set(`festival:${festival.toLowerCase()}_${year}:days`, JSON.stringify(allDates), redis.print);
```

Comment:
I don't think we can have two different types of objects saved for this key. 
Above in the .txt case it's saving a stringified list of the day numbers, i.e. "[1,2,3]" (same as the previous behavior). Here in the yaml case, it's saving a stringified list of the actual date string supplied from the yaml file, i.e. "["2022-01-01", "2022-01-02", ...]". 

I'd like to stick with just the day number representing days as much as possible, since it's like lowest common denominator. So the `allDates` list isn't necessary here, and day metadata (if available) will be accessed with the day number in the key, like you have it above already. You agree?